README.md

```diff
@@ -1,6 +1,17 @@
 # go-square
 
-`go-square` is a Go library designed to provide utility functions for data square operations.
+[![Go Reference](https://pkg.go.dev/badge/github.com/celestiaorg/go-square.svg)](https://pkg.go.dev/github.com/celestiaorg/go-square)
+
+`go-square` is a Go module that provides types and utilities for interacting with the original data square in a Celestia block.
+
+Package   | Description
+----------|---------------------------------------------------------------------------------------------------------------------
+blob      | Package blob provides types and functions for working with blobs, blob transactions, and index wrapper transactions.
+inclusion | Package inclusion contains functions to generate the blob share commitment from a given blob.
+merkle    | Package merkle computes a deterministic minimal height Merkle tree hash.
+namespace | Package namespace contains the Namespace data structure.
+shares    | Package shares contains the Share data structure.
+square    | Package square implements the logic to construct the original data square based on a list of transactions.
```

Comment:
Note to myself about making an issue about simplifying the pkg hierarchy. We don't need that many subpkgs.